src/engine/TPA.cc

```diff
@@ -1063,8 +1056,7 @@ bool TPASplit::checkExactFixedPoint(unsigned short power) {
 
 bool TPABase::verifyKinductiveInvariant(PTRef fla, unsigned long k) const {
     constexpr int trace_level = 1;
-    TRACE(trace_level, "Verifying k-inductive invariant for k = " << k)
-    {   // Inductive case:
+    TRACE(trace_level, "Verifying k-inductive invariant for k = " << k) { // Inductive case:
```

Comment:
This is incorrect change from clang-format. We need to make sure it does not do this.